compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/model/BIRNode.java

```diff
@@ -478,6 +489,11 @@ public void accept(BIRVisitor visitor) {
         public String toString() {
             return id.value;
         }
+
+        public void setId(int bbNumber) {
```

Comment:
It seems we don't use this method anywhere. Shall we remove it?

Reply:
Removed